systemtest/src/test/java/io/strimzi/systemtest/security/SecurityST.java

```diff
@@ -1328,6 +1335,148 @@ void testOwnerReferenceOfCASecrets() {
         });
     }
 
+    X509Certificate getCertificateFromSecret(Secret secret, String dataKey) {
+        String caCert = secret.getData().get(dataKey);
+        byte[] decoded = Base64.getDecoder().decode(caCert);
+        X509Certificate cacert = null;
+        try {
+            cacert = (X509Certificate)
+                    CertificateFactory.getInstance("X.509").generateCertificate(new ByteArrayInputStream(decoded));
+        } catch (CertificateException e) {
+            e.printStackTrace();
+        }
+        return cacert;
+    }
+
+    Secret getClusterSecret(String secretName) {
+        Secret secret = kubeClient().getSecret(CLUSTER_NAME_PREFIX + secretName, false);
+        if (secret == null) {
+            LOGGER.info("Systemwide secret not found. Using Cluster CAs validity {}",
+                    clusterName + "-" + CLUSTER_NAME_PREFIX + secretName);
+            secret = kubeClient().getSecret(clusterName + "-" + secretName, true);
+        }
+        LOGGER.debug("Using secret: {}", secret);
+        return secret;
+    }
```

Comment:
Out of curiosity -> when you don't find the secret in all namespaces, how you can find the secret in specific namespace?

Also this method should be in some `SecretUtils` class or something

Reply:
Seems like some strimzi secrets are project oriented, others are not. But I might be wrong on this one. @scholzj could clarify I believe. I will update accordingly.

Reply:
Removed whole method - not needed.